ooniprobe/Test/Test/AbstractTest.mm

```diff
@@ -275,8 +275,12 @@ -(void)writeString:(NSString*)str toFile:(NSString*)fileName{
 }
 
 -(void)onEntry:(JsonResult*)json obj:(Measurement*)measurement{
-    [self.result setStart_time:json.test_start_time];
-    [measurement setStart_time:json.test_start_time];
+    if (json.test_start_time != nil){
```

Comment:
@hellais this is the reason why `start_time` is not an invalid date

Reply:
I still don't see where you check the format of the string :|

Reply:
Again, it's not a string, it's a NSDate

Reply:
Where does the conversion from string inito NSDate happen?

Reply:
line 242